Helper/Webhook.php

```diff
@@ -282,7 +282,8 @@ private function updateNotification(Notification $notification, $processing, $do
             $notification->setDone(true);
         }
         $notification->setProcessing($processing);
-        $notification->setUpdatedAt(new DateTime());
+        $time = new DateTime();
+        $notification->setUpdatedAt($time->format('Y-m-d H:i:s'));
```

Comment:
In this case it seems acceptable just to use `date('Y-m-d H:i:s')` instead of instantiating an object